### DISCUSSION TITLE
Generating schema / interfaces from Models?

### DISCUSSION THREAD

**Discussion by popovicieduard** [adonisjs/core#2667] [General]
I have a question regarding tooling for frontend. Let's say I would want to generate interfaces from my adonis models, how would you approach this? I've looked into openapi-generator but couldn't find anything usefull.

**Comment by RomainLanz** [comment]
Hey @popovicieduard! 👋 

There is no way to generate a schema from your Models.
You can still have somehow an integration between your frontend and your backend.

Take a look at this example: https://github.com/mikealche/Next-Adonis-Typescript
